projects/clr/hipamd/src/hip_api_trace.cpp

```diff
@@ -2084,13 +2086,14 @@ HIP_ENFORCE_ABI(HipDispatchTable, hipLibraryLoadFromFile_fn, 497);
 HIP_ENFORCE_ABI(HipDispatchTable, hipLibraryUnload_fn, 498);
 HIP_ENFORCE_ABI(HipDispatchTable, hipLibraryGetKernel_fn, 499);
 HIP_ENFORCE_ABI(HipDispatchTable, hipLibraryGetKernelCount_fn, 500);
+HIP_ENFORCE_ABI(HipDispatchTable, hipStreamCopyAttributes_fn, 501);
 // if HIP_ENFORCE_ABI entries are added for each new function pointer in the table, the number below
 // will be +1 of the number in the last HIP_ENFORCE_ABI line. E.g.:
 //
 //  HIP_ENFORCE_ABI(<table>, <functor>, 8)
 //
 //  HIP_ENFORCE_ABI_VERSIONING(<table>, 9) <- 8 + 1 = 9
-HIP_ENFORCE_ABI_VERSIONING(HipDispatchTable, 501)
+HIP_ENFORCE_ABI_VERSIONING(HipDispatchTable, 502)
```

Comment:
This should never be changed without bumping the step version

Reply:
As far as I know, this ABI VERSIONING needs to be incremented for every new HIP API added.  And the HIP_RUNTIME_API_TABLE_STEP_VERSION is incremented only once for every new release.  For 7.2, this version is incremented to 15 already with the hipLibrary* API changes and hence all the new APIs will be under VERSION 15 itself until the next release.

Reply:
Updated the HIP_RUNTIME_API_TABLE_STEP_VERSION as discussed for this new API which has 2 stream arguments.